aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -100,19 +100,19 @@ The tutorial set you've just started covers these features of ASP.NET Web Pages:
 
 This tutorial set and subsequent ones revolve around a website where you can list movies that you like. You'll be able to enter movies, edit them, and list them. Here are a couple of the pages you'll create in this tutorial set. The first one shows the movie listing page that you'll create:
 
-![Finshed Movie app showing a movie listing](getting-started/_static/image1.png)
+![Screenshot of the My Movie Site application showing a movie listing, Search Genre and Search Title buttons, and search fields.](getting-started/_static/image1.png)
 
 And here's the page that lets you add new movie information to your site:
 
-![Finished movie app showing the Add Movie page](getting-started/_static/image2.png)
+![Screenshot of the finished movie app showing the Add Movie page with Title, Genre, and Year text fields.](getting-started/_static/image2.png)
 
 Subsequent tutorial sets build on this set and add more functionality, like uploading pictures, letting people log in, sending email, and integrating with social media.
 
 ## See this App Running on Azure
 
 Would you like to see the finished site running as a live web app? You can deploy a complete version of the app to your Azure account by simply clicking the following button.
 
-[![](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/?WT.mc_id=deploy_azure_aspnet&repository=https://github.com/tfitzmac/WebPagesMovies)
+[:::image type="icon" source="http://azuredeploy.net/deploybutton.png":::](https://azuredeploy.net/?WT.mc_id=deploy_azure_aspnet&repository=https://github.com/tfitzmac/WebPagesMovies)
```

Comment:
Don't change format, just add new alt text.

Reply:
Alt text was modified to reflect Microsoft’s accessibility guidelines for alt text per https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main. The accessibility guidelines also state that decorative images and icons, such as the deploy button icon, do not use include alt text and must use the format change proposed here.